public/request/game/update-image.php

```diff
@@ -14,9 +14,17 @@
 $input = Validator::validate(request()->post(), [
     'game' => 'required|integer|exists:mysql_legacy.GameData,ID',
     'type' => ['required', 'string', Rule::in(ImageType::cases())],
-    'file' => 'image',
+    'file' => ['image'],
 ]);
 
+if ($input['type'] === ImageType::GameIcon) {
+    Validator::make(
+        request()->all(),
+        ['file' => ['dimensions:width=96,height=96']],
+        ['file.dimensions' => 'Game icons are required to have dimensions of 96x96 pixels.']
```

Comment:
For future reference: we'd like to have any user-facing text to be translatable.
The default, translatable, validation error would read `The :attribute has invalid image dimensions.` if i'm not mistaken.
I think it's fine to go with that and add a (translatable) message below the input field that tells that images should be of said dimensions, file type, etc?
I tend to not add to many variants of custom validation errors as it becomes hard to maintain quickly.